javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,15 @@
+const fullName = 'Nhu Quan (Jimmy) Tran';
+const isCool = true;
+const totalPets = 0;
+console.log('full name: ', fullName);
+console.log('type of full name: ', typeof fullName);
+console.log('is kool? ', isCool);
+console.log('type of isCool: ', typeof isCool);
+console.log('total pets: ', totalPets);
+console.log('type of totalPets: ', typeof totalPets);
+const unknown = null;
+console.log('unknown variable: ', unknown);
+console.log('type of unknown: ', typeof unknown);
+const never = undefined;
```

Comment:
This should be using the `let` keyword, and you shouldn't be assigning any value to this variable.

Reply:
Got it. I have corrected the code: 
![image](https://user-images.githubusercontent.com/67088234/236959097-0f578e81-f316-4fbb-a232-0645dbf93c20.png)